tst/EnergyPlus/unit/UnitHeater.unit.cc

```diff
@@ -0,0 +1,1400 @@
+		"    UnitHeatAvailability,    !- Availability Schedule Name",
+		"    Zone2UnitHeatAirInletNode,  !- Air Inlet Node Name",
+		"    Zone2UnitHeatAirOutletNode, !- Air Outlet Node Name",
+		"    Fan:ConstantVolume,      !- Supply Air Fan Object Type",
+		"    Zone2UnitHeatFan,        !- Supply Air Fan Name",
+		"    0.47055,                 !- Maximum Supply Air Flow Rate {m3/s}",
+		"    Coil:Heating:Water,      !- Heating Coil Object Type",
+		"    Zone2UnitHeatHeatingCoil,!- Heating Coil Name",
+		"    ,                        !- Supply Air Fan Operating Mode Schedule Name",
+		"    No,                      !- Supply Air Fan Operation During No Heating",
+		"    7.25956E-005,            !- Maximum Hot Water or Steam Flow Rate {m3/s}",
+		"    0.0,                     !- Minimum Hot Water or Steam Flow Rate {m3/s}",
+		"    0.001;                   !- Heating Convergence Tolerance",
+
+		"  Fan:ConstantVolume,",
+		"    Zone2UnitHeatFan,        !- Name",
+		"    UnitHeatAvailability,    !- Availability Schedule Name",
+		"    0.5,                     !- Fan Total Efficiency",
+		"    75.0,                    !- Pressure Rise {Pa}",
+		"    0.47055,                 !- Maximum Flow Rate {m3/s}",
+		"    0.9,                     !- Motor Efficiency",
+		"    1.0,                     !- Motor In Airstream Fraction",
+		"    Zone2UnitHeatAirInletNode,  !- Air Inlet Node Name",
+		"    Zone2UnitHeatFanOutletNode;  !- Air Outlet Node Name",
+
+		"  Coil:Heating:Water,",
+		"    Zone2UnitHeatHeatingCoil,!- Name",
+		"    UnitHeatAvailability,    !- Availability Schedule Name",
+		"    84.93609,                !- U-Factor Times Area Value {W/K}",
+		"    7.25956E-005,            !- Maximum Water Flow Rate {m3/s}",
+		"    Zone2UnitHeatHWInletNode,!- Water Inlet Node Name",
+		"    Zone2UnitHeatHWOutletNode,  !- Water Outlet Node Name",
+		"    Zone2UnitHeatFanOutletNode, !- Air Inlet Node Name",
+		"    Zone2UnitHeatAirOutletNode, !- Air Outlet Node Name",
+		"    UFactorTimesAreaAndDesignWaterFlowRate,  !- Performance Input Method",
+		"    6750.70521,              !- Rated Capacity {W}",
+		"    82.2,                    !- Rated Inlet Water Temperature {C}",
+		"    16.6,                    !- Rated Inlet Air Temperature {C}",
+		"    71.1,                    !- Rated Outlet Water Temperature {C}",
+		"    32.2,                    !- Rated Outlet Air Temperature {C}",
+		"    ;                        !- Rated Ratio for Air and Water Convection",
+	} );
+	ASSERT_FALSE( process_idf( idf_objects ) );
+
+	NumOfTimeStepInHour = 4; // must initialize this to get schedules initialized
+	MinutesPerTimeStep = 15; // must initialize this to get schedules initialized
+	ProcessScheduleInput(); // read schedule data
+
+	ErrorsFound = false;
+	HeatBalanceManager::GetZoneData( ErrorsFound ); // read zone data
+	EXPECT_FALSE( ErrorsFound );
+
+	ErrorsFound = false;
+	GetZoneEquipmentData();
+	EXPECT_FALSE( ErrorsFound );
+
+	ErrorsFou
```

Comment:
Although this looks OK for now, could this ever result in an infinite loop in the future?